code/implementation/docker/machine/src/main/java/io/cattle/platform/docker/machine/launch/CatalogLauncher.java

```diff
@@ -29,15 +30,18 @@ protected String binaryPath() {
     }
 
     @Override
-    protected DynamicStringProperty getReloadSetting() {
-        return CATALOG_URL;
+    protected List<DynamicStringProperty> getReloadSettings() {
+        List<DynamicStringProperty> list = new ArrayList<DynamicStringProperty>();
+        list.add(CATALOG_URL);
+        list.add(CATALOG_REFRESH_INTERVAL);
+        return list;
     }
 
     @Override
     protected void prepareProcess(ProcessBuilder pb) {
         List<String> args = pb.command();
         args.add("-catalogUrl");
-        args.add(CATALOG_URL.get());
+        args.add("\"" + CATALOG_URL.get() + "\"");
```

Comment:
@prachidamle you shouldn't need to quote the string.  The nature of program arguments is that they are passed as a whole string.